read-skipfile-results

```diff
@@ -0,0 +1,433 @@
+    skipfile = yaml.load(skipfile_txt)
+
+    cleaned_skiplist = []
+
+    # Remove all_boards if used
+    for skipitem in skipfile["skiplist"]:
+        new_skipitem = deepcopy(skipitem)
+        cleaned_skiplist.append(new_skipitem)
+
+    skipfile["skiplist"] = cleaned_skiplist
+
+    # fix formatting and create a commit for it
+    with open(skipfile_name, "w") as output_yaml:
+        yaml.dump(skipfile, output_yaml)
+
+    repo.git.add(u=True)
+
+    github_repo = g.get_repo(f"{username}/test-definitions")
+
+    # Base patch - base patch for PRs
+    base_patch = "origin/master"
+
+    # If there are any formatting updates, make these
+    if repo.index.diff("HEAD"):
+        summary = "Skipfile formatting updates"
+        message = "Clean up skipfile formatting."
+        base = "master"
+        head = f"formatting-update-{date_str}-{patch_count}"
+        base_patch = head
+
+        logger.debug(
+            "branches:" + "\n".join([str(b.name) for b in github_repo.get_branches()])
+        )
+
+        create_commit_push_pr(
+            my_api_key,
+            repo,
+            github_repo,
+            username,
+            summary,
+            message,
+            base,
+            head,
+            args.github_push,
+        )
+
+        # Increment the patch counter
+        patch_count += 1
+        # Update the text version of skipfile
+        skipfile_txt = Path(skipfile_name).read_text()
+
+    run_info = ""
+    devices = set()
+
+    # fetch the run info
+    with open(args.metadata_filename, "r") as file:
+        for line in file:
+            # for each non-empty line
+            if line.strip():
+                (
+                    reproducer_script_name,
+                    run_project,
+                    device,
+                    git_desc,
+                ) = line.strip().split(",")
+                run_info += f"\n- {run_project}: {device}, git desc: {git_desc}"
+                devices.add(device)
+
+    test_dict = dict()
+    test_lookup = dict()
+
+    # loop through squad builds (one for each linux project tested)
+    for build_name in squad_builds:
+        logger.info(f"SQUAD build name {build_name}")
+
+        if build_name != "":
+            # Get the build whose name matches the build name from our list
+            build = first(project.builds(version=build_name))
+            # Look at the tests for that build
+            tests = build.tests()
+            # Check each of the tests
+            for test in tests.values():
+                # We only care about the custom commands not boot
+                if "commands" in test.name:
+                    testrun = TestRun(getid(test.test_run))
+                    get_file(testrun.job_url + "/results")
+                    reproducer = open("results", "r")
+                    results_json = load(reproducer)
+                    try:
+                        # Get names of tests run and split it into a list
```

Comment:
This shouldn't be needed if we push the command_name as just 'fork13' instead of
'cd /opt/ltp && ./runltp -s fork13'